src/modules/dashboard/components/Incorporation/IncorporationForm/Protectors/Protectors.tsx

```diff
@@ -0,0 +1,232 @@
+import React, { useCallback, useMemo } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { FieldArray, useField } from 'formik';
+import classNames from 'classnames';
+import { isEmpty } from 'lodash';
+import { nanoid } from 'nanoid';
+
+import { FormSection, InputLabel } from '~core/Fields';
+import QuestionMarkTooltip from '~core/QuestionMarkTooltip';
+import { AnyUser, Colony, useMembersSubscription } from '~data/index';
+import Icon from '~core/Icon';
+import UserPickerWithSearch from '~core/UserPickerWithSearch';
+import { filterUserSelection } from '~core/SingleUserPicker';
+import { supRenderAvatar } from '~dashboard/ExpenditurePage/Recipient/Recipient';
+import { Protector } from '~pages/IncorporationPage/types';
+import Button from '~core/Button';
+
+import Radio from '../Radio';
+import { SignOption } from '../types';
+import SingleUserPicker from '../SingleUserPicker';
+
+import styles from './Protectors.css';
+
+export const MSG = defineMessages({
+  protectorsLabel: {
+    id: `dashboard.Incorporation.IncorporationForm.Protectors.protectorsLabel`,
+    defaultMessage: 'Nominate protectors <span>(max 5)</span>',
+  },
+  protectorsTooltip: {
+    id: `dashboard.Incorporation.IncorporationForm.Protectors.protectorsTooltip`,
+    defaultMessage: `A Protector's role in a DAO legal corporation is to ratify the decisions of the DAO. Their purpose is to act on behalf of the DAO and handle legal the required administration. Learn more`,
```

Comment:
Please update to align with this comment - https://github.com/JoinColony/colonyDapp/pull/4137#discussion_r1093660088